boreas/util_tests.c

```diff
@@ -57,6 +61,12 @@ __wrap_socket (__attribute__ ((unused)) int domain,
 
 bool g_setsockopt_use_real = true;
 int
+__wrap_setsockopt (__attribute__ ((unused)) int sockfd,
```

Comment:
Can you tell me, why you duplicated this? Just for my information? :)

Reply:
To declare it so that it has a prototype. Since it is just needed for tests I didn't want to declare a header and I find it better than #pragma